### PR TITLE
debian: fix snapd.refresh.service install and usage

### PR DESCRIPTION
**Pull request by mvo5**
A mismatch in the debiian/snapd.install rule for the refresh time/service. Plus removal of the old name "snappy-autopilot.{service,timer}".

Closes: LP: #1588977

### CODE REVIEW

**Comment by niemeyer** [comment]
LGTM. Integration tests are failing.. hiccup?

**Comment by elopio** [comment]
The failure is one of the limitations we get because we can't create all-snaps images from branches. The service in the test bed is still named snappy-autopilot.timer, so you get an error when you try to stop the unexisting snapd.refresh.timer. 
The old name will remain until a new version is pushed to edge and we can generate a new image from it. The autopkgtests will fail if you leave the old name, so I suggest you replace the two statements in the base test with:

exec.Command("sudo", "systemctl", "stop", "snapd.refresh.timer").Run()
exec.Command("sudo", "systemctl", "disable", "snapd.refresh.timer").Run()
// XXX old name kept to get the all-snaps tests working until a new core version is released to edge
exec.Command("sudo", "systemctl", "stop", "snappy-autopilot.timer").Run()
exec.Command("sudo", "systemctl", "disable", "snappy-autopilot.timer").Run()

The error is not nice because it comes from the init of the test modules. We shouldn't do anything in the init, but there's a struggle between repeating the same set up on every suite (which is slow) and doing it only once (which can lead to inaccurate errors)

**Comment by elopio** [comment]
The autopkgtest failed because:
https://launchpad.net/~godeps-maintainers/godeps/trunk/.bzr/branch-format: Unable to handle http code 503: Service Temporarily Unavailable
It seems to be working now.

**Comment by elopio** [comment]
retest this please

**Comment by mvo5** [comment]
Here is the output from this morning on my laptop with this new package:
```
$ systemctl status snapd.refresh.service
● snapd.refresh.service - Automatically refresh installed snaps
   Loaded: loaded (/lib/systemd/system/snapd.refresh.service; static; vendor pres
   Active: inactive (dead) since Sa 2016-06-04 08:37:35 CEST; 1min 4s ago
  Process: 7089 ExecStart=/usr/bin/snap refresh (code=exited, status=0/SUCCESS)
 Main PID: 7089 (code=exited, status=0/SUCCESS)

Jun 04 08:37:30 bod systemd[1]: Starting Automatically refresh installed snaps...
Jun 04 08:37:35 bod snap[7089]: Name         Version               Rev  Developer
Jun 04 08:37:35 bod snap[7089]: hello-world  6.1                   26   canonical
Jun 04 08:37:35 bod snap[7089]: ubuntu-core  16.04+20160531.11-56  122  canonical
Jun 04 08:37:35 bod systemd[1]: Started Automatically refresh installed snaps.
```

File: debian/snapd.refresh.service [hunk 1]
```diff
@@ -1,8 +1,9 @@
 [Unit]
 Description=Automatically refresh installed snaps
 After=network.target
+Documentation=man:snap(1)
```
Comment (zyga) [inline debian/snapd.refresh.service]: Nice touch, thank you

**Comment by mvo5** [comment]
retest this please

**Pull request merged**